src/main/groovy/com/inxmail/clicktracker/controller/RedirectController.groovy

```diff
@@ -0,0 +1,19 @@
+package com.inxmail.clicktracker.controller
+
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.ResponseBody
+
+@Controller
+class RedirectController {
+    @RequestMapping(value="/{slug}", method = RequestMethod.GET)
+    @ResponseBody
+    public ResponseEntity<Void> redirect (@PathVariable("slug") String slug) {
+        ResponseEntity.status(HttpStatus.FOUND).header("Location","https://www.google.de/?gws_rd=ssl#q="+slug).build()
```

Comment:
Please use a HTTP Status 301 - this seems to be the more correct version for these kinds of redirects.

All the big players (t.co, goo.gl, bitly.com, ...) seem to use 301 as well.